libraries/core-react/src/Table/Cell.jsx

```diff
@@ -34,18 +61,33 @@ const borderTemplate = (borders) =>
     return `${acc} border-${val}: ${width} solid ${color}; \n`
   }, '')
 
-const Base = ({ tokens }) => {
+const Base = ({ tokens, as }) => {
   const { background, height, text, spacings, borders } = tokens
   const { typography } = text
+  const bordersAndBackground =
+    as === 'th'
+      ? `
+        thead & {
+          ${borderTemplate(borders.thead)}
+          background: ${background.thead};
+        }
+
+        tbody & {
+          ${borderTemplate(borders.tbody)}
+          background: ${background.tbody};
+        }`
+      : `
+        ${borderTemplate(borders)}
+        background: ${background}`
+
```

Comment:
Add `css` function from `styled-components` for better readability?

Reply:
That wouldn’t work in this case

Reply:
```jsx
css`
thead & {
    ${borderTemplate(borders.thead)}
    background: ${background.thead};
}

tbody & {
    ${borderTemplate(borders.tbody)}
    background: ${background.tbody};
}`
```

css has support for string templates no?

Reply:
The output from that is 
```css
thead &{, border-bottom: 2px solid rgba(220, 220, 220, 1); 
, background:,rgba(247, 247, 247, 1),;}tbody &{, border-bottom: 1px solid rgba(220, 220, 220, 1); 
, background:,rgba(255, 255, 255, 1),;}
```

Reply:
Don’t think the css helper is appropriate to use here